### PR TITLE
reverseBits

### PR DESCRIPTION
**Pull request by peinanteng**

### CODE REVIEW

File: reverseBits.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Solution:
+    # @param n, an integer
+    # @return an integer
+    def reverseBits(self, n):
+        ans = 0
+        for i in range(32):
+            ans = (ans << 1) + (n & 1)
```
Comment (nkukarl) [inline reverseBits.py]: Why do you need `()`?

File: reverseBits.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Solution:
+    # @param n, an integer
+    # @return an integer
+    def reverseBits(self, n):
+        ans = 0
+        for i in range(32):
```
Comment (nkukarl) [inline reverseBits.py]: Can do `for _ in range(32):`

File: reverseBits.py (hunk 1 above)
Comment (peinanteng) [reply reverseBits.py]: if no (), the output for 1 will be zero. but the correct answer should be   2147483648 (10000000000000000000000000000000).

File: reverseBits.py (hunk 2 above)
Comment (peinanteng) [reply reverseBits.py]: good suggestion

**Pull request merged**